### PR TITLE
Added profile of Suhani Agarwal

### LINKED ISSUE #2155: [Community] Member Profile: Suhani Agarwal
**Current Behavior**

@suhaniii  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @suhaniii  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @suhaniii 
- Twitter: 
- LinkedIn: suhani-agarwal (https://www.linkedin.com/in/suhani-agarwal/)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

The CONTRIBUTING.md file has a "Common Types of Site Updates" section with detailed examples to follow here.[CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by vishal-codes**
**Description**
Added profile of Suhani Agarwal on Community members page of website.
This PR fixes #2155

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
Overall LGTM🎉, let's see the preview once a maintainer approves the workflow

**Review by suhaniii — APPROVED** [review]
LGTM 🎊 Thank you @warunicorn19 @vishal-codes

**Review by Tanuj22 — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 78e11909105fe9a448bfa72c133a1f43e5b2c006 at: https://61361591fece2523eff78fac--layer5ng.netlify.app

**Pull request merged**
